src/databricks/labs/ucx/recon/base.py

```diff
@@ -64,13 +65,19 @@ class SchemaComparisonResult:
     is_matching: bool
     data: list[SchemaComparisonEntry]
 
+    def as_dict(self):
+        return dataclasses.asdict(self)
+
 
 @dataclass
 class DataComparisonResult:
     source_row_count: int
     target_row_count: int
-    num_missing_records_in_target: int
-    num_missing_records_in_source: int
+    num_missing_records_in_target: int = 0
+    num_missing_records_in_source: int = 0
```

Comment:
```suggestion
    source_missing_count: int = 0
```
be consistent with units everywhere